analysis/model/make_model_input.R

```diff
@@ -70,8 +74,18 @@ for (i in 1:nrow(active_analyses)) {
                            unlist(strsplit(active_analyses$covariate_other[i], split = ";")),#[!grepl("_priorhistory_",unlist(strsplit(active_analyses$covariate_other[i], split = ";")))],
```

Comment:
Remove commented out code as not used anymore and in the Git history.

Reply:
Done.